Update installation.md (#830)
falcosecurity/falco-website: content/en/docs/getting-started/installation.md

    ```bash
    yum erase -y falco
    ```

>  > _Note_: If you are using UEFI enabled systems please run:

Reviewer: ```suggestion
 > You might need to validate the driver signature if your system has UEFI SecureBoot enabled.\
> Follow these steps to do so:
>
```

>  >         sudo mokutil --import /var/lib/dkms/mok.pub

Reviewer: ```suggestion
 >         1. Import the DKMS Machine Owner Key
 >         $ sudo mokutil --import /var/lib/dkms/mok.pub
 >
```

>  >         Restart the system (A MOK key enrollment will prompt)

Reviewer: ```suggestion
 >         2. Restart the system and wait for the MOK key enrollment prompt
 >
```

>  >         Enroll MOK

Reviewer: ```suggestion
 >         3. Choose the option:  Enroll MOK
 >
```

>  >         insmod /var/lib/dkms/falco/4.0.0+driver/$(uname -r)/x86_64/module/falco.ko.xz     

Reviewer: ```suggestion
 >         4. Load the Falco driver
 >         $ insmod /var/lib/dkms/falco/4.0.0+driver/$(uname -r)/x86_64/module/falco.ko.xz     
```
